demosplan/DemosPlanCoreBundle/DoctrineMigrations/2023/01/Version20230104090531.php

```diff
@@ -0,0 +1,129 @@
+<?php declare(strict_types = 1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Exception;
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+class Version20230104090531 extends AbstractMigration
+{
+    private const NEW_G_NAME = 'basemap';
+    private const NEW_G_LAYER = 'de_basemapde_web_raster_farbe';
+    private const NEW_G_URL = 'https://sgx.geodatenzentrum.de/wms_basemapde';
+    //private const G_PROJECTION_LABEL = 'EPSG:3857';
+    //private const G_PROJECTION_VALUE = '+proj=merc +a=6378137 +b=6378137 +lat_ts=0.0 +lon_0=0.0 +x_0=0.0 +y_0=0 +k=1.0 +units=m +nadgrids=@null +wktext +no_defs';
+    private const NEW_COPYRIGHT_TEXT = '© basemap.de BKG (www.basemap.de) / LVermGeo SH (www.LVermGeoSH.schleswig-holstein.de)';
```

Comment:
According to https://sgx.geodatenzentrum.de/web_public/gdz/lizenz/deu/basemapde_web_dienste_lizenz.pdf this needs to be `© basemap.de / BKG 2023`

Reply:
Was answered in Ticket: 
`© basemap.de BKG (www.basemap.de) / LVermGeo SH (www.LVermGeoSH.schleswig-holstein.de)`
is exactly what is requested.